src/main/docker/configs/bucketeer.properties.default

```diff
@@ -26,6 +26,7 @@ HTTP_MAX_QUEUE_SIZE=5
 S3_UPLOADER_INSTANCES=1
 S3_UPLOADER_THREADS=20
 S3_MAX_REQUESTS=10
+S3_MAX_RETRIES=20
```

Comment:
You set this as 30 in the pom, but 20 here...

Reply:
Updated for consistency, thanks.